src/routes/safe/components/Transactions/helpers/EditTxParametersForm/index.tsx

```diff
@@ -170,73 +164,63 @@ export const EditTxParametersForm = ({
-                      name="ethNonce"
-                      defaultValue={ethNonce}
-                      placeholder="Nonce"
-                      text="Nonce"
-                      type="number"
-                      component={TextField}
-                      disabled={!areEthereumParamsVisible(parametersStatus)}
-                    />
-                    <Field
-                      name="ethGasLimit"
-                      defaultValue={ethGasLimit}
-                      placeholder="Gas limit"
-                      text="Gas limit"
-                      type="number"
-                      component={TextField}
-                      disabled={parametersStatus === 'CANCEL_TRANSACTION'}
-                    />
-                    {((gasPriceText) => (
-                      <Field
-                        name="ethGasPrice"
-                        defaultValue={ethGasPrice}
-                        type="number"
-                        placeholder={gasPriceText}
-                        text={gasPriceText}
-                        component={TextField}
-                        disabled={!areEthereumParamsVisible(parametersStatus)}
-                      />
-                    ))(`${isMaxFeeParam() ? 'Max fee per gas' : 'Gas price'} (GWEI)`)}
-
-                    {isMaxFeeParam() && (
-                      <Field
-                        name="ethMaxPrioFee"
-                        defaultValue={ethMaxPrioFee}
-                        type="number"
-                        placeholder="Max priority fee (GWEI)"
-                        text="Max priority fee (GWEI)"
-                        component={TextField}
-                        disabled={!areEthereumParamsVisible(parametersStatus)}
-                      />
-                    )}
-                  </EthereumOptions>
-
-                  <StyledLink
-                    href="https://help.gnosis-safe.io/en/articles/4738445-configure-advanced-transaction-parameters-manually"
-                    target="_blank"
-                  >
-                    <Text size="xl" color="primary">
-                      How can I configure these parameters manually?
-                    </Text>
-                    <Icon size="sm" type="externalLink" color="primary" />
-                  </StyledLink>
-                </>
-              )}
+              <EthereumOptions>
+                <Field
+                  name="ethNonce"
+                  defaultValue={ethNonce}
+                  placeholder="Nonce"
+                  text="Nonce"
+                  type="number"
+                  component={TextField}
+                />
+                <Field
+                  name="ethGasLimit"
+                  defaultValue={ethGasLimit}
+                  placeholder="Gas limit"
+                  text="Gas limit"
+                  type="number"
+                  component={TextField}
+                  disabled={parametersStatus === 'CANCEL_TRANSACTION'}
```

Comment:
Do we not have an enum for this?

Reply:
We don't atm, and you're commenting on something that was already there.

Reply:
Alright. It's still good to improve whilst we can though